tests/integration/pages/desktop/scanResults.page.js

```diff
@@ -1,6 +1,3 @@
-/* global $$ */
-/* global $ */
```

Comment:
These were already declared globally for integration tests in .eslintrc overrides, so this was getting flagged as a redeclare. :shrug: